.travis.yml

```diff
@@ -241,7 +246,8 @@ script:
   - ./autogen.sh
   - CONFIGURE_FLAGS="$ASCIIDOC_OPTIONS $COVERAGE_OPTIONS $HARDENING_OPTIONS $MODULES_OPTIONS $NSS_OPTIONS $OPENSSL_OPTIONS $RUST_OPTIONS --enable-fatal-warnings --disable-silent-rules"
   - echo "Configure flags are $CONFIGURE_FLAGS CC=\"$CC $C_DIALECT_OPTIONS\""
-  - ./configure $CONFIGURE_FLAGS CC="$CC $C_DIALECT_OPTIONS"
+  - if [[ "$ALL_BUGS_ARE_FATAL" == "" ]]; then ./configure $CONFIGURE_FLAGS CC="$CC $C_DIALECT_OPTIONS"; fi
+  - if [[ "$ALL_BUGS_ARE_FATAL" != "" ]]; then ./configure CFLAGS="-DALL_BUGS_ARE_FATAL" $CONFIGURE_FLAGS CC="$CC $C_DIALECT_OPTIONS"; fi
```

Comment:
Using `CFLAGS` like this seems to replace all the other default cflags that we use -- Maybe we should have it be `-DALL_BUGS_ARE_FATAL -O2 -g` for now?  Or maybe we can fix up CFLAGS handling in our configure script

Reply:
Addressed in a0741a2a3821171f9aeb9928804df6eef1e4f172.